docs/db-schema.md

```diff
@@ -51,8 +51,8 @@ homedomain | VARCHAR(44) | (BASE64)
 thresholds | TEXT | (BASE64)
 flags | INT NOT NULL |
 lastmodified | INT NOT NULL | lastModifiedLedgerSeq
-buyingliabilities | BIGINT CHECK (buyingliabilities >= 0) |
-sellingliabilities | BIGINT CHECK (sellingliabilities >= 0) |
+extension | TEXT | Opaque XDR extension specific to AccountEntry
```

Comment:
Nitpick: for consistency with the rest of the file these should read like "Extension specific to AccountEntry (XDR)". See the legend at the top of this file.

Reply:
Done.